src/Microsoft.AspNetCore.Cors/Internal/CORSLoggerExtensions.cs

```diff
@@ -64,6 +66,16 @@ static CORSLoggerExtensions()
                 LogLevel.Warning,
                 9,
                 "Failed to apply CORS Response headers.");
+
+            _noCorsPolicyFound = LoggerMessage.Define(
+                LogLevel.Information,
+                new EventId(10, "NoCorsPolicyFound"),
+                "No CORS policy found for the specified request.");
+
+            _insecureConfiguration = LoggerMessage.Define(
+                LogLevel.Warning,
+                new EventId(11, "CorsInsecureConfiguration"),
+                "CORS policy uses an insecure configuration that allows any origin and supports credentials.");
```

Comment:
cc @Eilon for text of the log

Reply:
What exactly is this message trying to say? Is it recommending that the user not do this? If so, it doesn't really seem to say what the effective problem is, nor how to fix it.

Reply:
> CORS protocol does not allow specifying a wildcard (any) origin and credentials. Configure the policy by listing individual origins if credentials needs to be supported.

Does that make more sense?

Reply:
How about:

> The CORS protocol does not allow specifying a wildcard (any) origin and credentials at the same time. Configure the policy by listing individual origins if credentials needs to be supported.